sqlalchemy_schemadisplay.py

```diff
@@ -213,13 +213,19 @@ def create_schema_graph(tables=None, metadata=None, show_indexes=True, show_data
     return graph
 
 def show_uml_graph(*args, **kwargs):
-    from cStringIO import StringIO
+    try:
+        from cStringIO import StringIO as sIO   # for python 2.7
```

Comment:
Is it really necessary to differentiate between StringIO and BytesIO? BytesIO is available in Python 2.7. I haven't tried this code yet, so maybe it is necessary for Python 2 due to some other part later on.

Reply:
I am not changing the beahvior for python 2.7. All test pass for python 2.7. BytesIO is only imported in case of python 3.x.

Reply:
@mayanksuman just use [`BytesIO`](https://docs.python.org/2.7/library/io.html#io.BytesIO) for both Python 2.x and Python 3.x

Reply:
@abitrolly I have used appropriate `StringIO` in the latest commit.